src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift

```diff
@@ -47,11 +51,36 @@ class HomeStatisticsCellModel {
 		localStatisticsProvider.remove(region)
 	}
 
+	func filterKeyFigures(keyFigures: [SAP_Internal_Stats_KeyFigureCard]) -> [SAP_Internal_Stats_KeyFigureCard] {
+		if isCombinedIncidenceCardSupported {
+			return keyFigures.filter { $0.header.cardID != 2 && $0.header.cardID != 8 }
+		} else {
+			return keyFigures.filter { $0.header.cardID != 10 }
+		}
+	}
+
 	// MARK: - Private
 
 	private let homeState: HomeState
 	private let localStatisticsProvider: LocalStatisticsProviding
 
 	private var subscriptions = Set<AnyCancellable>()
-
+	private var isCombinedIncidenceCardSupported: Bool {
+		#if DEBUG
+		if isUITesting {
+			return !LaunchArguments.statistics.useMockedOldAppVersion.boolValue
+		} else if isOldAppVersion {
+			return false
+		}
+		#endif
+		// we show the combined card 10 only for 2.13 or later, else we show 2 and 8
```

Comment:
Won't the version for this delivery will always the 2.13? In which scenario can we end up with this code and an app version < 2.12?

Reply:
I just updated the code